lib/rules/web/ocm_console/dialogs_and_sidebar.xyaml

```diff
@@ -257,3 +257,55 @@ check_persistent_storage_field_in_scale_dialog:
       xpath: //label[@for='persistent_storage']/span[text()='Persistent storage']
   - selector:
       xpath: //select[@name='persistent_storage']
+choose_loadbalancer_in_dialog:
```

Comment:
How about control the `choose_loadbalancer_in_dialog` and `choose_persistent_quota_in_dialog` and node scale with one action and parameter. For example:
select_persistent_quota_in_dialog:
    action: click_persistent_quota_dropdown
    element:
        xpath: <xpath>
        op: click
scale_cluster:
  action: 
      if_para: persistent_quota_number
       ref: select_persistent_quota_in_dialog
  action: 
      if_param: <LB param>
      ref: choose_loadbalancer_in_dialog
...
  action: click_apply_button
  action: check_dialog_missing